P/picotool/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "picotool"
+version = v"1.1.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/raspberrypi/picotool.git", "f6fe6b7c321a2def8950d2a440335dfba19e2eab"),
+    GitSource("https://github.com/raspberrypi/pico-sdk.git", "6a7db34ff63345a7badec79ebea3aaef1712f374")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/picotool/
+cp udev/99-picotool.rules $prefix/99-picotool.rules
+install_license LICENSE.TXT
+mkdir build
+cd build
+PICO_SDK_PATH=$WORKSPACE/srcdir/pico-sdk cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_EXE_LINKER_FLAGS="-static-libgcc -static-libstdc++" -DCMAKE_BUILD_TYPE=Release ..
```

Comment:
You need to pass the toolchain file: https://docs.binarybuilder.org/stable/build_tips/#CMake-builds

Reply:
Not sure what that should have helped with - `musl` breaks anyway https://buildkite.com/julialang/yggdrasil/builds/5984#018b2f3c-9e98-45ff-a3f7-a7835b0a3b1e/662-842

Reply:
I've removed musl again, since I don't really have a use for that. If someone with a musl system wants to use this, we can always debug it later.